src/System.Management.Automation/CoreCLR/CorePsPlatform.cs

```diff
@@ -561,20 +561,255 @@ internal static int NonWindowsGetProcessParentPid(int pid)
         // the string perror would give you for the last set value of errno.
```

Comment:
@JamesWTruher, your last commit had 1 failures in `PowerShell-CI-linux`
Get-FormatData.Can get format data requiring '-PowerShellVersion 5.1'
```powershell
Expected a collection with size 4, but got collection with size 5 @(System.Management.Automation.FormatViewDefinition, System.Management.Automation.FormatViewDefinition, System.Management.Automation.FormatViewDefinition, System.Management.Automation.FormatViewDefinition, System.Management.Automation.FormatViewDefinition).
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/Get-FormatData.Tests.ps1: line 18
18:         $format.FormatViewDefinition | Should -HaveCount 4
```